atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CommitTsLoader.java

```diff
@@ -38,6 +39,8 @@
 public final class CommitTsLoader {
```

Comment:
This class should really just use a LoadingCache internally so that we have eviction, I'm happy to do that in a separate PR

Reply:
Yes. Will be coming in part 2